hack/certification/certify.sh

```diff
@@ -0,0 +1,116 @@
+#!/bin/bash
+# This script runs Openshift preflight against Calico images and optionally uploads the results to Openshift Connect for the certification process.
+# In addition, the certification process also requires the following, which are NOT handled by this script:
+# - the operator metadata bundle (see the tigera/operator repo) to be submitted by PR
+#   to https://github.com/redhat-openshift-ecosystem/certified-operators
+# - running the Openshift End-to-End tests - we run them for CNI and CNV.  The results need to be passed to the Certification team for verification.
+
+set -x
```

Comment:
Can we use `set -e` to fail on errors?  Or does that not make sense for this script?

Reply:
Preflight exits with 1 for failed tests.  I really want it to continue and test the other images.  

I guess I could have added `set -ex` here and then `|| true` for the preflight tests only.  

But that's 90% of this script, so I felt it was simpler not to do that.